charts/fluentd-elasticsearch/templates/configmaps.yaml

```diff
@@ -10,6 +10,11 @@ data:
   system.conf: |-
     <system>
       root_dir /tmp/fluentd-buffers/
+      {{- if .Values.fluentdLogFormat }}
```

Comment:
imho we can remove the if clause and always set the default

Reply:
👍